.github/workflows/build-test.yml

```diff
@@ -94,3 +94,27 @@ jobs:
                     ./configure --with-v8js=/opt/v8/self-built LDFLAGS="-lstdc++" CPPFLAGS="-DV8_COMPRESS_POINTERS -DV8_ENABLE_SANDBOX"
                     make
                     make test
+
+    alpine:
+        runs-on: ubuntu-latest
```

Comment:
Is this how one runs alpine on GitHub Workflows? :D weird concept but if it works

Reply:
yeah, found it weird as well :)

... but there's no runner image with alpine